### ISSUE TITLE
Python 3 only Project Documentation with autodoc.

### ISSUE THREAD

**Issue by avinashs95** [rtfd/readthedocs.org#3467]
## Details

* Project URL: https://readthedocs.org/projects/sprw-iot/
* Build URL (if applicable): https://readthedocs.org/projects/sprw-iot/builds/
* Read the Docs username (if applicable): avinash_sprw

## Expected Result

Python3 project should be installed in readthedocs environment. API Documentation should become available.

## Actual Result

> /home/docs/checkouts/readthedocs.org/user_builds/sprw-iot/checkouts/latest/src/docs/sprw.rst:14: WARNING: autodoc: failed to import module 'sprw'; the following exception was raised:
> Traceback (most recent call last):
>   File "/home/docs/checkouts/readthedocs.org/user_builds/sprw-iot/envs/latest/lib/python3.5/site-packages/sphinx/ext/autodoc.py", line 658, in import_object
>     __import__(self.modname)
> ImportError: No module named 'sprw'

Since the my project package is not installed, its unable to import the module.
Only the index page is properly displayed. [http://sprw-iot.readthedocs.io/en/latest/](url). If navigated to other links, it shows 

```
   \          SORRY            /
         \                         /
          \    This page does     /
           ]   not exist yet.    [    ,'|
           ]                     [   /  |
           ]___               ___[ ,'   |
           ]  ]\             /[  [ |:   |
           ]  ] \           / [  [ |:   |
           ]  ]  ]         [  [  [ |:   |
           ]  ]  ]__     __[  [  [ |:   |
           ]  ]  ] ]\ _ /[ [  [  [ |:   |
           ]  ]  ] ] (#) [ [  [  [ :===='
           ]  ]  ]_].nHn.[_[  [  [
           ]  ]  ]  HHHHH. [  [  [
           ]  ] /   `HH("N  \ [  [
           ]__]/     HHH  "  \[__[
           ]         NNN         [
           ]         N/"         [
           ]         N H         [
          /          N            \
         /           q,            \
        /                           \

```
I specified the requirements.txt file in my readthedocs project dashboard settings. But it's getting installed using default python version 2 and throws error while installing the requirements.

> 
> python /home/docs/checkouts/readthedocs.org/user_builds/sprw-iot/envs/latest/bin/pip install --exists-action=w --cache-dir /home/docs/checkouts/readthedocs.org/user_builds/sprw-iot/.cache/pip -rsrc/requirements.txt
> Collecting actionlib==1.11.9 (from -r src/requirements.txt (line 1))
>   Could not find a version that satisfies the requirement actionlib==1.11.9 (from -r src/requirements.txt (line 1)) (from versions: )
> No matching distribution found for actionlib==1.11.9 (from -r src/requirements.txt (line 1))

How to overcome this issue?

**Comment by stsewd** [comment]
I can't find `actionlib` package on pypi https://pypi.python.org/pypi/actionlib.

Also there is no results on `pip3 search actionlib` and `pip2 seach actionlib`.

**Comment by avinashs95** [comment]
My Bad. I generated requirements.txt using pip freeze. But it had generated the packages present in the whole of my python environment. Then I used pipreqs to generate requirements specific to my project. Package installation is now fine. 

Regarding the page does not exist, I added this in conf.py, 
```
import os
import sys
sys.path.insert(0, os.path.abspath('../../'))
```
This is because my folder structure is **/src/docs/source/conf.py**. 
**src** contains the source files of my python module.
